### PR TITLE
Multi language titles

### PR DESCRIPTION
**Pull request by angelk**
Fix for https://github.com/trackhub/website/issues/108

### CODE REVIEW

File: translations/messages.bg.php [hunk 1]
```diff
@@ -7,7 +7,13 @@
     'Log in' => 'Вписване',
     'Name' => 'Име',
     'Type' => 'Тип',
+    'Submit' => 'Изпрати',
     'GPS tracks by track-hub.com' => 'GPS следи от track-hub.com',
+    'Name in Bulgarian' => 'Име на Български',
+    'Name in English' => 'Име на Английски',
+    'Visibility' => 'Видимист',
```
Comment (StefanMavrodiev) [inline translations/messages.bg.php]: "Видимист"

File: translations/messages.bg.php [hunk 2]
```diff
@@ -7,7 +7,13 @@
...
+    'public' => 'публичен',
+    'unlisted' => 'нелистнат',
```
Comment (StefanMavrodiev) [inline translations/messages.bg.php]: "нелистнат"
I don't know that word

File: translations/messages.bg.php (hunk 1 above)
Comment (angelk) [reply translations/messages.bg.php]: fixed

File: translations/messages.bg.php (hunk 2 above)
Comment (angelk) [reply translations/messages.bg.php]: Fixed

**Comment by angelk** [comment]
> Can we use other variable names? Something like name_bg_bg, name_en_us, etc. In case more languages are supported?


I don't think we should have such detailed manguages.
For example CRC have only

- en
- es
- fr
- it
- pt
- ru
- ja


> We can make the English title mandatory and the other one is depending on the locale?
Probably we will have users with no English knowledge. I prefer to (later) add translations manually instead of having non senses.

**Pull request merged**